rust/private/clippy.bzl

```diff
@@ -141,12 +141,11 @@ def _clippy_aspect_impl(target, ctx):
         args.process_wrapper_flags.add("--stderr-file", clippy_out)
 
         if clippy_flags:
-            fail("""Combining @rules_rust//:clippy_flags with @rules_rust//:capture_clippy_output=true is currently not supported.
-See https://github.com/bazelbuild/rules_rust/pull/1264#discussion_r853241339 for more detail.""")
+            args.rustc_flags.add_all(clippy_flags)
```

Comment:
Could you ping that discussion thread to mention that this PR lifts this restriction?